examples/jsm/renderers/CSS2DRenderer.js

```diff
@@ -114,8 +115,21 @@ class CSS2DRenderer {
 
 		};
 
+		function hideObject( object ) {
+
+			if ( object.isCSS2DObject ) object.element.style.display = "none";
+
+			for ( let i = 0, l = object.children.length; i < l; i++ ) {
+
+				hideObject( object.children[i] );
+
+			}
+		}
+
 		function renderObject( object, scene, camera ) {
 
+			if ( object.visible === false ) return;
```

Comment:
It seems the ` object.visible === true ) &&` further below can be removed now.

Reply:
That's right. The object has to be visible === true below, so there's no need to check again.
Thus, line 138 can be
```diff
- const visible = ( object.visible === true ) && ( _vector.z >= - 1 && _vector.z <= 1 ) && ( object.layers.test( camera.layers ) === true );
+ const visible = ( _vector.z >= - 1 && _vector.z <= 1 ) && ( object.layers.test( camera.layers ) === true );
```

Reply:
Beautiful!